openwisp_radius/base/models.py

```diff
@@ -197,7 +198,25 @@ def clean(self):
             )
 
 
-class AbstractRadiusCheck(OrgMixin, AutoUsernameMixin, TimeStampedEditableModel):
+class UniqueAtrributeMixin(object):
+    def clean(self):
+        """
+        checks if the check or reply attribute is unique
+        """
+        model = type(self).__name__
+        if 'RadiusGroup' in model:
+            options = dict(group=self.group)
+        else:
+            options = dict(organization=self.organization, user=self.user)
+        options.update(attribute=self.attribute)
+        if load_model(model).objects.filter(**options).exclude(pk=self.pk).exists():
+            raise ValidationError({'attribute': _('This attribute is already in use.')})
```

Comment:
The error message should be improved.

If it's a group check/reply should be:
`Another {object_name} for the same group and with the same attribute already exists`.

Otherwise:
`Another {object_name} for the same user and with the same attribute already exists`.

Where object_name is either group check, group reply, check or reply.

Also in this case, the error message could be returned by a method, eg:

eg:

```python
GroupAttributeValidationMixin(AttributeValidationMixin):
    def __get_validation_queryset_kwargs(self):
        return dict(group=self.group)

    def __get_error_message(self):
        object_name = .....  # find out whether it's check or a reply
        return f'Another {object_name} for the same group and with the same attribute already exists'  
```